src/mds/nameserver2/curvefs.cpp

```diff
@@ -280,6 +282,25 @@ StatusCode CurveFS::CreateFile(const std::string & fileName,
         }
     }
 
+    std::vector<PoolIdType> logicalPools =
+                topology_->GetLogicalPoolInCluster();
+    std::map<PoolIdType, double> enoughSpace;
+    bool hasEnough = false;
+    chunkSegAllocator_->GetRemainingSpaceInLogicalPool(&logicalPools,
```

Comment:
The return value of `chunkSegAllocator_->GetRemainingSpaceInLogicalPool` is not used. So this function `GetRemainingSpaceInLogicalPool` can directly return void